extension/tests/onboarding.test.ts

```diff
@@ -0,0 +1,144 @@
+import { shuffle } from "lodash";
```

Comment:
do you mind changing this dir name to `e2eTests` so we can differentiate between these and unit tests?

Reply:
yup can do

Reply:
renamed in e99194053f8fb93000911ce5cb43589f3ef43894